torch/csrc/throughput_benchmark/init.cpp

```diff
@@ -0,0 +1,63 @@
+#include <ATen/core/ivalue.h>
+#include <torch/csrc/throughput_benchmark/ThroughputBenchmark.h>
+#include <torch/csrc/utils/auto_gil.h>
+#include <torch/csrc/utils/pybind.h>
+#include <torch/csrc/jit/pybind_utils.h>
+
+
+#include <pybind11/functional.h>
+
+namespace torch {
+namespace throughput_benchmark {
+
+void initThroughputBenchmarkBindings(PyObject* module) {
+  auto m = py::handle(module).cast<py::module>();
+  using namespace torch::throughput_benchmark;
+  py::class_<BenchmarkConfig>(m, "BenchmarkConfig")
+      .def(py::init<>())
+      .def_readwrite(
+          "num_calling_threads", &BenchmarkConfig::num_calling_threads)
+      .def_readwrite("num_worker_threads", &BenchmarkConfig::num_worker_threads)
+      .def_readwrite("num_warmup_iters", &BenchmarkConfig::num_warmup_iters)
+      .def_readwrite("num_iters", &BenchmarkConfig::num_iters);
+
+  py::class_<ExecutionStats>(m, "ExecutionStats")
+      .def_readonly("latency_avg_ms", &ExecutionStats::latency_avg_ms)
+      .def_readonly("num_iters", &ExecutionStats::num_iters)
+      .def_readonly("iters_per_second", &ExecutionStats::iters_per_scond);
+
+  py::class_<ThroughputBenchmark>(m, "ThroughputBenchmark", py::dynamic_attr())
+      .def(py::init<std::shared_ptr<jit::script::Module>>())
+      .def(
+          "add_input",
+          [](py::args args, py::kwargs kwargs) {
```

Comment:
you can make the first argument ThroughputBenchmark& directly - from docs:

> You may also use just one or the other, and may combine these with other arguments as long as the py::args and py::kwargs arguments are the last arguments accepted by the function.

Reply:
did it work?

Reply:
yep